src/ResourceManager/Websites/AzureRM.Websites.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.2.2'
+ModuleVersion = '5.0.0'
```

Comment:
What version do we want this module to be? Having a preview and a stable module with the same version could be confusing and might cause problems for installation - do we want this to bump to 6.0.0?